### PR TITLE
Introduce Buefy

### PR DESCRIPTION
**Pull request by exbinary**
Also some cleanup of `application.scss`

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,44 +1,17 @@
...
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
```
Comment (exbinary) [inline app/assets/stylesheets/application.scss:15]: @maebeale you were so right about this! My bad 🙇 .

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -1,44 +1,17 @@
...
 .text-center {
   text-align: center;
 }
-.main-footer {
-  position: absolute;
-  bottom: 0;
-  width: 100%;
-  height: 60px;
-  line-height: 60px;
-}
```
Comment (exbinary) [inline app/assets/stylesheets/application.scss:30]: Wasn't used.

File: app/assets/stylesheets/custom_spacing_helpers.scss [hunk 3]
```diff
@@ -16,4 +16,4 @@ $positions: ('top', 'left', 'bottom', 'right');
       padding-#{$position}: ($size + .5) + rem;
     }
   }
-}
\ No newline at end of file
+}
```
Comment (exbinary) [inline app/assets/stylesheets/custom_spacing_helpers.scss]: Looks like one or more of us have a newline-on-last line configuration discrepancy.
I have mine setup to always include a newline, as advised [here](https://thoughtbot.com/blog/no-newline-at-end-of-file) and other places.

Comment (maebeale) [reply app/assets/stylesheets/custom_spacing_helpers.scss]: i don't have a setting about it, but have started to add them by habit bc my coworker's linter is always adding them. happy to do whatever anyone wants!

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (maebeale) [reply app/assets/stylesheets/application.scss:15]: haha! np!!!

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -1,44 +1,17 @@
...
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+@import 'font-awesome';
 
-$input-placeholder-color: hsl(0, 0%, 86%); //has-text-grey-lighter
```
Comment (maebeale) [inline app/assets/stylesheets/application.scss:17]: i think i still need this on the rails side. it's a bulma variable so my placeholder text isn't black in rails forms

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -1,44 +1,17 @@
...
-}
-.select {
-  vertical-align: baseline;
-}
-.form-actions {
-  padding-top: 3em;
-}
-.select:not(.is-multiple):not(.is-loading)::after, .navbar-link:not(.is-arrowless)::after {
-  margin-top: 2.5em;
-}
```
Comment (maebeale) [inline app/assets/stylesheets/application.scss:44]: i'm still using all of these in rails simple_form forms

File: app/assets/stylesheets/bulma_customization.scss [hunk 6]
```diff
@@ -0,0 +1 @@
+$input-placeholder-color: hsl(0, 0%, 86%); //has-text-grey-lighter
```
Comment (maebeale) [inline app/assets/stylesheets/bulma_customization.scss:1]: ah. i see. sorry! shouldn't have commented as i went!

File: app/javascript/entry_points/publicListings.js [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import Vue from 'vue'
+import Vue from 'our_vue'
```
Comment (maebeale) [inline app/javascript/entry_points/publicListings.js:1]: maybe `buefy_vue` ?

Comment (exbinary) [reply app/javascript/entry_points/publicListings.js:1]: Yeah i don't love the name `our_vue`, but we'll likely further configure our Vue instance with other plugins and such.

**Pull request merged**